### PR TITLE
fix (DeBank): Add missing enum members

### PR DESCRIPTION
**Pull request by hanjano**

### CODE REVIEW

File: blockapi/v2/models.py [hunk 1]
```diff
@@ -40,12 +45,23 @@ class AssetType(Enum):
...
     LENDING_REWARD = 'lending-reward'
+    LIQUIDITY_POOL = 'liquidity pool'
     REWARDS = 'rewards'
     COMMON = 'common'
+    FARMING = 'farming'
     LOCKED = 'locked'
     YIELD = 'yield'
 
 
+DEBANK_BLOCKCHAIN = {
```
Comment (galvanizze) [inline blockapi/v2/models.py:56]: what about the other blockchains? conversion for them is not needed?

File: blockapi/v2/models.py [hunk 2]
```diff
@@ -31,10 +31,25 @@ class Blockchain(Enum):
     ASTAR = "astar"
     IOTEX = "iotex"
 
-    XDAI = 'xdai'
-    OKT = 'okt'
-    HARMONY = 'hmy'
-    MATIC = 'matic'
+    AURORA = 'aurora'
```
Comment (hanjano) [inline blockapi/v2/models.py:34]: Is the order of Blockchain enum somehow important, or may I sort it?

Comment (galvanizze) [reply blockapi/v2/models.py:34]: no it's not..

note: I just remembered every EVM (ethereum virtual machine) blockchain has its own `ChainID`, maybe we should work somehow with it to easier map tokens with currencies.. https://chainlist.org

**Review by galvanizze — APPROVED** [review]

File: blockapi/v2/models.py (hunk 2 above)
Comment (galvanizze) [reply blockapi/v2/models.py:34]: ha! https://openapi.debank.com/v1/chain/list `community_id == chain_id` 😎 nice

**Pull request merged**